### ISSUE TITLE
AVT 2 & 3 - Overflow Menu has JAWS screen reader issues

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4361]
## Environment
Microsoft Windows 10 w/VMWare Infusion
Firefox Version 67
JAWS Version 19.1810.64

## Description
1. With JAWS running Tab to the Overflow Menu
2. With the Default or Trigger: Use the Space key to open the menu. The menu opens and then disappears. In addition, the Space key and Enter keys close the menu.
**Expected Result:** The menu should remain open until the Esc key is pressed. 
Also note: That the issue is only present using the Space key to open the menu. In some cases you may need to open the menu a couple of times to see the issue.  This is not an issue with the Trigger version of the component or when using the Enter key. 
3. Navigate down the first menu options list using the Arrow keys. The visual focus stops at Option 3. This only happens on Windows using only the keyboard as well as using the keyboard with JAWS. 
**Expected result:** Visual focus should be on the focused menu item.

**Comment by snidersd** [comment]
@dakahn This issue was also reported by Josiah Hoskins on macOS when trying to navigate. He stated, "So on a Mac in the ACR pages when the OverflowMenu is tabbed to nothing happens. A tab will take you to the menu (the three vertical dots) and a tab will take you away. However, the menu does not open and allow one to see or hear the options using tab (or arrows)."

**Comment by asudoh** [comment]
Here's what I know; When overflow menu gets open, typically the focus is put on the first item. When user hits space key or enter key after then, it means the menu is selected and thus the menu will be closed. So by "The menu should remain open until the Esc key is pressed.", wondering if space key not selecting the menu item is desirable, or menu not being closed upon selection update is desirable, or something else.

**Comment by dakahn** [comment]
![Jan-20-2020 11-26-57](https://user-images.githubusercontent.com/40970507/72746507-e22d7b80-3b77-11ea-8f7c-557c05b8f6e9.gif)

Testing with JAWS on Windows 10 in Firefox and I can't reproduce the menu closing unintentionally after opening with spacebar or the missing focus indicators when traversing the menu options. So seems we're good there 👍 

@asudoh our menu options are "buttons" and so the user expectation would be activation with either enter or spacebar. I think we're fine there according to the [APG](https://www.w3.org/TR/wai-aria-practices-1.1/#keyboard-interaction-3) though _buttons in a menu_ may have a different expection.

@snidersd I'll close this one up -- but feel free to comment if I've misread/missed something or if buttons in a menu are handled differently keyboard wise and I can reopen ❤️
